src/generators.rs

```diff
@@ -327,10 +342,10 @@ fn gen_pow(scale: u16) -> (Option<String>, String) {
         body.push_str(&*format!("(pow u{} u{}) ", n1, n2));
     }
 
-    (None, body)
+    GenOutput::new(None, body, 2)
```

Comment:
Are you just using 2 (instead of 1) to denote a small constant?

Reply:
yep. using 2 because `pow` takes two args